asteroidsyncservice/watch.h

```diff
@@ -52,19 +56,24 @@ public slots:
 signals:
     void weatherCityNameChanged();
     void batteryLevelChanged();
+    void timeServiceChanged();
 
 private:
     QVariant fetchProperty(const QString &propertyName);
 
 private slots:
     void dataChanged();
+    void tServiceReady();
```

Comment:
Please keep explicit names. "tService" isn't clear

Reply:
i switch to "timeServiceReady"